include/dynd/types/fixed_string_type.hpp

```diff
@@ -17,7 +17,8 @@ namespace ndt {
 
   class DYNDT_API fixed_string_type : public base_string_type {
     intptr_t m_stringsize;
-    string_encoding_t m_encoding;
+    const string_encoding_t m_encoding;
+    const std::string m_encoding_repr;
```

Comment:
Is there a reason to store a copy of this in here? I think it's probably nicer to just have the enum to keep things lighter weight.